GitUI/Editor/FileViewer.cs

```diff
@@ -607,14 +607,12 @@ public Task ViewFileAsync(string fileName, bool isSubmodule = false, Action? ope
             {
                 try
                 {
-                    var path = _fullPathResolver.Resolve(fileName);
-
-                    if (!File.Exists(path))
+                    if (!File.Exists(fullPath))
```

Comment:
Let's add an assert
```suggestion
                    Debug.Assert(Path.IsPathFullyQualified(path), "Path must be resolved and fully qualified");
                    if (!File.Exists(fullPath))
```

Reply:
```suggestion
                    if (!File.Exists(fullPath))
                    Debug.Assert(Path.IsPathFullyQualified(fullPath), "Path must be resolved and fully qualified");
```

Reply:
No, we want to assert before checking file.

Reply:
Made a minor refactoring of the handling with properly ordered checks (a few lines removed)